daemon/cluster/convert/swarm.go

```diff
@@ -12,6 +13,14 @@ import (
 
 // SwarmFromGRPC converts a grpc Cluster to a Swarm.
 func SwarmFromGRPC(c swarmapi.Cluster) types.Swarm {
+	var defaultAddrPool []*net.IPNet
+	for _, address := range c.DefaultAddressPool {
+		_, b, err := net.ParseCIDR(address)
+		if err == nil {
```

Comment:
Looks like we're ignoring errors here; should we handle invalid address-pools here?

Reply:
I see a similar comment was made above https://github.com/moby/moby/pull/37558#discussion_r211017334; if we deliberately ignore those errors, that may warrant a comment here to explain that

Reply:
will add comment